docs/03-policies/prodev.md

```diff
@@ -0,0 +1,102 @@
+# Professional Development at CivicActions
+
+## Introduction
+
+CivicActions recognizes the importance of individual Professional Development for all team members. We value Professional Development for a number of reasons:
+
+* It’s a win-win. Professional Development grows the depth and value of the company offerings and enhances your career opportunities too.
+* Professional Development provides career mobility for each person - increasing our feeling of safety, which leads to:
+    * Having the courage to fail, which leads to experimentation, innovation, and learning
+    * Having the courage to challenge the status quo, which leads to organizational learning
+    * Having the courage to be vulnerable, which leads to authentic communications and connection
+    * Open-minded thinking, which leads to increased tolerance of others ideas and personality styles
+
+CivicActions supports and encourages the Professional Development (prodev) of each team member by providing an annual budget of $1,200 per person to be used for prodev-related expenses.
+
+In addition to the $1,200 annual budget, each team member is encouraged to use a “take what you need” approach to using time for prodev pursuits during business hours. If you need guidance on what amount of time is appropriate to spend on prodev in a given year, consult the table below.
```

Comment:
I wonder if there should be a statement in here about potential impact to billable work. If someone takes what they need and it negatively impacts a deliverable, I would hope this could be a red flag or would warrant someone pinging their PM upfront to request this time when it's least disruptive.